### PR TITLE
chore: use Message for no data in ComparisonView

### PR DESCRIPTION
**Pull request by johnkim-det**
## Description

In the Compare panel on the Experiment List page, each of the three tabs ("Metrics", "Hyperparameters", "Details") have different empty state messages. Use the same message for all of them ("No data available.")

Screen Shot 2023-08-23 at 4 35 11 PM

## Test Plan

Select experiments for comparison that do not have trial data. Make sure all three tabs display the "No data available." message.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

WEB-1512

### CODE REVIEW

File: webui/react/src/components/kit/LineChart.tsx [hunk 1]
```diff
@@ -439,7 +442,13 @@ export const ChartGrid: React.FC<GroupProps> = React.memo(
                     columnCount={columnCount}
                     columnWidth={Math.floor(width / columnCount)}
                     height={height - 40}
-                    itemData={{ chartsProps: chartsProps, columnCount, handleError, scale, xAxis }}
+                    itemData={{
+                      chartsProps: chartsProps,
```
Comment (keita-determined) [inline webui/react/src/components/kit/LineChart.tsx:446]: nit: just `chartsProps`

**Review by keita-determined — APPROVED** [review]
LGTM

**Comment by ClaireNeveu** [comment]
Pretty sure this should be a Message and not an Alert.

**Comment by johnkim-det** [comment]
> Pretty sure this should be a Message and not an Alert.

I know we've said in the past that Message should be used for indicating an empty state. However, in the UI kit PRD work, we proposed a different guideline for Message vs. Alert usage (page-level vs. component-level). That's still somewhat of an open discussion, but I figured it made sense to go with Alert here for now since we also use it for other similar messages within the Compare panel. That was my thinking, but happy to go with Message for this ticket if that's preferred.

**Comment by johnkim-det** [comment]
will go ahead and use Message here

**Review by keita-determined — APPROVED** [review]
LGTM

**Pull request merged**